vulcanai/run.py

```diff
@@ -0,0 +1,47 @@
+from torch.utils.data import DataLoader, Subset
+from vulcanai.datasets import TabularDataset
+from vulcanai.models import ConvNet, DenseNet
+import sys
+sys.path.append('../')
+from models.metrics import conduct_sensitivity_analysis
+
+def my_test_dataset():
+    """Create a dataset by importing from the test csv"""
+    fpath = "/Users/snehadesai/Documents/Vulcan/vulcanai/tests/datasets/test_data/birthweight_reduced.csv"
```

Comment:
I would use a relative path depending on the actual package and not your installation of Vulcan